### DISCUSSION TITLE
How to open the Properties ?

### DISCUSSION THREAD

**Discussion by DigitalAge3000** [kangyu-california/PersistentWindows#202] [Q&A] [answered]
Hi,

I'm running v5.40 but cannot find a way to open the properties as shown in the screenshot. How can I access the settings ?

Thanks !

![grafik](https://user-images.githubusercontent.com/123281961/213898512-220b26c7-9b81-4ef0-8f87-76918d808844.png)

**Comment by kangyu-california** [comment, marked as answer]
double click a task (in task scheduler) will bring up the properties dialog
